src/weather_gen.cpp

```diff
@@ -225,7 +227,7 @@ weather_type weather_generator::get_weather_conditions( const w_point &w ) const
 int weather_generator::get_wind_direction( const season_type season, unsigned seed ) const
 {
     unsigned dirseed = seed;
-    cata_default_random_engine wind_dir_gen( dirseed );
+    static cata_default_random_engine wind_dir_gen( dirseed );
```

Comment:
This means the seed is essentially on any subsequent call. But this may get called with a new seed when a new game is started or loaded.
```suggestion
    cata_default_random_engine wind_dir_gen( dirseed );
```

Reply:
Without that static, then it gets passed the same seed, and therefore wind never changes direction.

Reply:
I see what you mean: each invocation (while playing the same game) uses the same seed, so the random engine is in the same state each time.

But my argument still stands: loading a second save will carry over the random engine created from within the first game and will thereby *ignore* the seed of the second game.

Btw. what's the point of copying the function parameter `seed` into a local variable and using that one? Why not use the function parameter directly?

Reply:
Seeing as the initial wind PR that added that code, over a year ago,  was one of my first PRs, and I was even more inexperienced than I am now, I can only  guess as to the reasons behind not using the function parameter directly, and I think the answer is " I have no idea what I was doing" , so I will change that. I will also think of a solution that solves the random engine seed problem.

Reply:
Is it suitable to use the time_since_epoch based seed for the random engine? this was originally the case when I first added wind direction, then I passed the game seed down to it in a subsequent PR, for reasons I cannot remember.